src/Native/Bootstrap/main.cpp

```diff
@@ -372,6 +372,11 @@ extern "C" void RhpBulkWriteBarrier()
 {
     throw 42;
 }
+
+extern "C" Object* RhNewArrayAsString(MethodTable*)
```

Comment:
This comes from the following code generated:

System_Private_CoreLib::System::String* System_Private_CoreLib::System::String::FastAllocateString(int32_t length){System_Private_CoreLib::System::String* _l0=0; _bb0: {
if (!__statics.__cctor_System_Private_CoreLib__System__String) { __statics.__cctor_System_Private_CoreLib__System__String = true; System_Private_CoreLib::System::String::_cctor(); }void* _1=__gcStatics.System_Private_CoreLib__System__String__Empty; 
System_Private_CoreLib::System::EETypePtr _2=System_Private_CoreLib::System::Object::get_EETypePtr((System_Private_CoreLib::System::Object*)_1); 
int32_t _4=length; 
void* _5=System_Private_CoreLib::System::Runtime::RuntimeImports::RhNewArrayAsString(_2, _4); 

int32_t _7=length; 
((System_Private_CoreLib::System::String*)_5)->_stringLength=_7; 
_l0=(System_Private_CoreLib::System::String*)_5; 
} {goto _bb33; } _bb26: {
} _bb33: {
void* _9=_l0; 
return (System_Private_CoreLib::System::String*)_9; } }